molgenis-navigator/src/main/resources/l10n/navigator_nl.properties

```
@@ -8,4 +8,7 @@ table-col-header-description=Omschrijving
 delete-confirmation-text=Weet je zeker dat je deze items permanent wilt verwijderen?
 delete-confirmation-title=Bevestig Verwijderen
 delete-confirmation-ok-text=Verwijder
-delete-confirmation-cancel-text=Annuleer
\ No newline at end of file
+delete-confirmation-cancel-text=Annuleer
+progress-download-running=Begonnen met voorbereiden van de download...
+progress-download-success=Klaar met voorbereiden van de download.
+NAV01=Download gefaald: {0}
```

```suggestion
N03=Download gefaald: {0}
```
To harmonize with Copy Exceptions